esp/src/src-react/components/Result.tsx

```diff
@@ -251,20 +257,14 @@ export const Result: React.FunctionComponent<ResultProps> = ({
             .logicalFile(logicalFile)
             .filter(filter)
             .renderHtml(renderHTML)
-            .lazyRender()
+            .renderPromise().then(() => setResult(resultTable.calcResult()))
```

Comment:
Hmmm I forgot that renderPromise even existed - for consistency it might be better to use:

```
render(() => setResult(resultTable.calcResult());
```

Which should be the same thing, but I would like to double check the renderPromise code...